src/jit/emitxarch.cpp

```diff
@@ -13663,6 +13663,1128 @@ size_t emitter::emitOutputInstr(insGroup* ig, instrDesc* id, BYTE** dp)
+                result.insThroughput = PERFSCORE_THROUGHPUT_1C;
+            }
+            break;
+
+#ifdef _TARGET_AMD64_
+        case INS_movsxd:
+#endif
+        case INS_mov:
+        case INS_movsx:
+        case INS_movzx:
+        case INS_cmp:
+        case INS_test:
+            if (memFmt == IF_NONE)
+            {
+                result.insThroughput = PERFSCORE_THROUGHPUT_4X;
+            }
+            else if (memAccessKind == PERFSCORE_MEMORY_READ)
+            {
+                result.insThroughput = PERFSCORE_THROUGHPUT_2X;
+            }
+            else // writes
+            {
+                assert(memAccessKind == PERFSCORE_MEMORY_WRITE);
+                result.insThroughput = PERFSCORE_THROUGHPUT_1C;
+            }
+            break;
+
+        case INS_adc:
+        case INS_sbb:
+            result.insLatency = max(PERFSCORE_LATENCY_1C, result.insLatency);
+            if (memAccessKind == PERFSCORE_MEMORY_READ_WRITE)
+            {
+                result.insThroughput = PERFSCORE_THROUGHPUT_1C;
+            }
+            else
+            {
+                result.insThroughput = PERFSCORE_THROUGHPUT_2X;
+            }
+            break;
+
+        case INS_add:
+        case INS_sub:
+        case INS_and:
+        case INS_or:
+        case INS_xor:
+            result.insLatency = max(PERFSCORE_LATENCY_1C, result.insLatency);
+            if (memFmt == IF_NONE)
+            {
+                result.insThroughput = PERFSCORE_THROUGHPUT_4X;
+            }
+            else if (memAccessKind == PERFSCORE_MEMORY_READ_WRITE)
+            {
+                result.insThroughput = PERFSCORE_THROUGHPUT_1C;
+            }
+            else
+            {
+                result.insThroughput = PERFSCORE_THROUGHPUT_2X;
+            }
+            break;
+
+        case INS_lea:
+            // uops.info
+            result.insThroughput = PERFSCORE_THROUGHPUT_1C;
+            result.insLatency    = PERFSCORE_LATENCY_1C;
+            break;
+
+        case INS_imul_AX:
+        case INS_imul_BX:
+        case INS_imul_CX:
+        case INS_imul_DX:
+        case INS_imul_BP:
+        case INS_imul_SI:
+        case INS_imul_DI:
+#ifdef _TARGET_AMD64_
+        case INS_imul_08:
+        case INS_imul_09:
+        case INS_imul_10:
+        case INS_imul_11:
+        case INS_imul_12:
+        case INS_imul_13:
+        case INS_imul_14:
+        case INS_imul_15:
+#endif // _TARGET_AMD64_
+        case INS_mulEAX:
+        case INS_imulEAX:
+        case INS_imul:
+            result.insThroughput = PERFSCORE_THROUGHPUT_1C;
+            result.insLatency    = PERFSCORE_LATENCY_3C;
+            break;
+
+        case INS_div:
+        case INS_idiv:
+            result.insThroughput = PERFSCORE_THROUGHPUT_6C;
+            // The integer divide instruction has a long latency
+            if ((id->idOpSize() == EA_8BYTE))
+            {
+                result.insLatency = PERFSCORE_LATENCY_50C;
```

Comment:
It looks like the throughput should also be updated for this path... Its 21-83 for DIV and 24-90 for IDIV

Reply:
OK, If you like I can use 52 for DIV and 57 for IDIV, which is the average value of (21..83) and (24..90)

Reply:
That's fine. I was mostly just noting that it isn't 6, as for the 1, 2, and 4-byte versions 😄

Reply:
I will update both throughput  and latency for DIV and IDIV